Source/NameServer/ABDefs.cpp

```diff
@@ -0,0 +1,54 @@
+#include "ABDefs.hpp"
```

Comment:
I can't really tell what "ABD" stands for here. Andrew Brown Dennis Elastic File System?

Edit: Oh, "AddressBook"! Please spell it out :(